hansol/db/migrations/20221102080531_create_users.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE users 
+(
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,       
+  email VARCHAR(200) NOT NULL,
+  profile_image VARCHAR(1000) NULL,    
+  password VARCHAR(200) NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),        
+  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP           
+); 
```

Comment:
한솔님, sql파일 관련하여, 쿼리문을 아래와 같이 수정해 주세요! 다른 파일들도 동일하게 수정 부탁드립니다 !
```suggestion
CREATE TABLE users (
  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  name VARCHAR(50) NOT NULL,       
  email VARCHAR(200) NOT NULL,
  profile_image VARCHAR(1000) NULL,    
  password VARCHAR(200) NOT NULL,
  created_at TIMESTAMP NOT NULL DEFAULT NOW(),        
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP           
); 
```